pkg/router/template/configmanager/haproxy/testing/haproxy.go

```
@@ -121,22 +118,18 @@ func (p *fakeHAProxy) Start() {
 			shutdown := p.shutdown
 			p.lock.Unlock()
 			if shutdown {
-				return nil
+				return
 			}
 			conn, err := listener.Accept()
 			if err != nil {
-				return err
+				return
 			}
 			go p.process(conn)
```

golangci-lint (2.12.2)

[error] Error return value of `p.process` is not checked

(errcheck)